File: ruby/yapay_gw/lib/yapay_gw/models/recurring_payment.rb
class RecurringPayment
    attr_accessor :recurring_payment_data
    attr_accessor :store_code

    def as_json(options = {})
        json = {:recorrencia => recurring_payment_data, 
                :estabelecimento => store_code
            }
        json.delete_if { |k, v| v.nil? }
        json
    end
end

File: net/YapayGatewayLib/model/RecurringPayment.cs
using System;
using Newtonsoft.Json;

namespace YapayGatewayLib.Model
{
    public class RecurringPayment
    {
        [JsonProperty(PropertyName = "recorrencia")]
        public RecurringPaymentData recurringPaymentData { get; set; }

        [JsonProperty(PropertyName = "estabelecimento")]
        public string storeCode { get; set; }

    }
}
